uv.lock

```diff
@@ -1385,15 +1385,15 @@ wheels = [
 
 [[package]]
 name = "sentry-sdk"
-version = "2.18.0"
+version = "2.13.0"
```

Comment:
this is the only package thats actually being downgraded here, and I would suggest that you not pin this, but rather define a lower bound, like for the others.

The other packages are not actually being downgraded in the lockfile, but rather their minimum version is being lowered, which has an effect on api/worker pulling in `shared` and having to resolve dependency versions. So in api/worker, we *could* end up with lower dependencies.

If you are hitting actual errors with deprecated/removed API, you can pin this to a lower version using `abs<x.y.z` syntax like for `sqlalchemy` in the `pyproject.toml`.
Not quite sure whether you want to do that here in this repo, or in api/worker rather.

Reply:
Good points - thanks!
I'll bump/fix the breaking change in sentry-sdk 2.18.0 after we get to a clean slate again - https://github.com/codecov/engineering-team/issues/2994